### ISSUE TITLE
DataCite CSL-JSON container-title

### ISSUE THREAD

**Issue by bwiernik** [zotero/translators#2048]
DataCite is currently returning "Series" information in `container-title` (e.g., https://data.datacite.org/application/citeproc+json/10.3886%2FICPSR35531.v3).

This should probably be stored in `collection-title` instead. I imagine this is probably best to be corrected in DataCite's API, rather than post cleanup in a translator?

**Comment by zuphilip** [comment]
[Just short]: I don't think this is wrong in general by DataCite, because all OJS journals using DataCite DOIs are pushing the journal information into this series field. CC @mfenner

**Comment by zuphilip** [comment]
See also https://support.datacite.org/docs/description

**Comment by bwiernik** [comment]
Oh, that’s really frustrating then. If the Series field may or may not contain an actual container title, it’s hard to get good consistent results from citation styles.

**Comment by mfenner** [comment]
The `SeriesInformation` metadata in DataCite are free-text, so really hard to parse. We have a "recommended" way, and use that when registering metadata via bibtex or RIS. And the next major DataCite metadata scheme (5.0, end of 2020) will handle container-title, volume, issue and page numbers properly.

**Comment by bwiernik** [comment]
Could collection-title also be added to the scheme to separate information like OJS from the types of information like I linked to above?

**Comment by mfenner** [comment]
The details of what will be included in DataCite Schema 5.0 have not been specified, but `collection-title` makes sense.
